### PR TITLE
Transactions

### PR DESCRIPTION
**Pull request by a-nurmatov**

### CODE REVIEW

File: client/src/app/category/components/category-delete-confirm/category-delete-confirm.component.scss [hunk 1]
```diff
@@ -30,5 +30,8 @@
 
 ::ng-deep .snackbar-error {
```
Comment (AudriusBaltusis) [inline client/src/app/category/components/category-delete-confirm/category-delete-confirm.component.scss:31]: please use `:host::ng-deep`

File: client/src/app/transaction/service/transaction.service.ts [hunk 2]
```diff
@@ -18,6 +18,16 @@ export class TransactionService {
...
   ) {}
 
+  getAllTransactions(): Observable<{
+    message: string;
+    transactions: TransactionInterface[];
+  }> {
+    return this.http.get<{
+      message: string;
+      transactions: TransactionInterface[];
+    }>('http://localhost:5000/transactions/all');
```
Comment (AudriusBaltusis) [inline client/src/app/transaction/service/transaction.service.ts:28]: Please create constant for `http://localhost:5000`

File: client/src/app/category/components/category-delete-confirm/category-delete-confirm.component.scss [hunk 3]
```diff
@@ -30,5 +30,8 @@
...
   background-color: var.$warningBg;
-  color: var.$textDanger !important;
+  color: var.$textLight !important;
+  button {
+    color: var.$textLight !important;
```
Comment (AudriusBaltusis) [inline client/src/app/category/components/category-delete-confirm/category-delete-confirm.component.scss:35]: Is `!important` needed here?

**Comment by a-nurmatov** [comment]
Hi, how are you?
[Ninja code](https://javascript.info/ninja-code)
After reading this article I realized what kind of mistakes I was making. Thank you for your support during this internship, and for correcting my mistakes.

**Comment by a-nurmatov** [comment]
I think you can not stop laughing if you read the above article :).

**Comment by AudriusBaltusis** [comment]
this article is old but gold :) Good luck in coding

**Pull request merged**